extensions/azure-keyvault/deployment/src/main/java/io/quarkiverse/azure/keyvault/secret/deployment/KeyVaultSecretProcessor.java

```diff
@@ -0,0 +1,107 @@
+package io.quarkiverse.azure.keyvault.secret.deployment;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.Type;
+import org.jboss.logging.Logger;
+
+import com.azure.json.JsonSerializable;
+
+import io.quarkiverse.azure.keyvault.secret.runtime.KeyVaultSecretClientProducer;
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.IndexDependencyBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageProxyDefinitionBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveHierarchyBuildItem;
+
+public class KeyVaultSecretProcessor {
+
+    static final String FEATURE = "azure-keyvault-secret";
+    private static final Logger log = Logger.getLogger(KeyVaultSecretProcessor.class.getName());
```

Comment:
how about using `io.quarkus.logging.Log` to simplify the usage.

Reply:
Quarkus uses the JBoss Log Manager logging backend for publishing application and framework logs. See https://quarkus.io/guides/logging.

Reply:
Thanks for the helpful comment @backwind1233.  I think the simple logging approach is good enough here, will change to `io.quarkus.logging.Log`.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/0db5e3b0a6afba790d5b5645d8751a18e9e2a5b9.